requirements/extra-ml-requirements.txt

```diff
@@ -20,6 +20,9 @@ tensorflow>=2.8.0
 torch>=1.11.0
 torchvision>=0.12.0
 pytorch_lightning>=1.5.10
+# Avoid version 1.9.0 of gyptorch, which is incompatible
+# with the versions of botorch used in many versions of pytorch
+gpytorch!=1.9.0
```

Comment:
I think we need to add this in `examples/pytorch/IterativePruning/conda.yaml` and `examples/pytorch/AxHyperOptimizationPTL/conda.yaml`.